jupyterlab/labapp.py

```diff
@@ -580,6 +580,14 @@ class LabApp(NotebookConfigShimMixin, LabServerApp):
         document will be kept in memory forever.""",
     )
 
+    collaborative_document_save_delay = Int(
```

Comment:
As per your comment in https://github.com/jupyterlab/jupyterlab/pull/13240 I agree that we should allow a float as this is compatible with the underlying `asyncio.sleep` call.

```suggestion
    collaborative_document_save_delay = Float(
```

Reply:
Should #13240 supersede this PR?